### DISCUSSION TITLE
[WASM] Where is WebAssemblyRuntime located?

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#7746] [Q&A]
Appears the reference 
`WebAssembly.WebAssemblyRuntime`
[here ](https://platform.uno/docs/articles/interop/wasm-javascript-1.html)is incorrect but should be 
`Uno.Foundation.WebAssemblyRuntime`

**Comment by jeromelaban** [comment]
Thanks for the report! This got fixed in latest master.
